index.bs

```diff
@@ -2107,20 +2124,16 @@ an |ownership type|, a |serialization internal map|, a |realm| and a |session|:
              1. If |shadow root| is null, let |serialized shadow| be null.
                 Otherwise run the following substeps:
 
-               1.  Let |child depth| be |max depth| - 1 if |max depth| is not
-                  null, or null otherwise.
-
                1. Let |serialized shadow| be the result of
-                  [=serialize as a remote value=] with |shadow root|, |child depth|,
-                  false, |ownership type|, |serialization internal map|,
+                  [=serialize as a remote value=] with |shadow root|,
+                  |serialization options|, |ownership type|, |serialization internal map|,
```

Comment:
Shouldn't we decrease `maxDomDepth` here?

Reply:
See the commit message. The intent is that we always serialize the id of the shadow root, but whether we descend into the actual nodes in the shadow tree depends on the other options that are specified.